### ISSUE TITLE
Request a new release

### ISSUE THREAD

**Issue by sehkone** [petabi/review-protocol#44]
A new release is required after #43 to release a new version of the review-web.

**Comment by msk** [comment]
I've reviewed the changes in #43, and I'm wondering why removing the config-related items is "required" for review-web. Could you provide more context or explain how these changes impact review-web? Specifically, what difference would it make if review-web just uses review-protocol-0.6.0 rather than a new version? I'd appreciate understanding the reasoning behind this requirement before proceeding with the release.

**Comment by sehkone** [comment]
The main reason for removing the config-related items is simply that they are no longer needed. The server no longer requests agents to retrieve their configs from the server. Instead, it requests them to update their configs. And, we now use TOML strings.

On the other hand, the reason why this is required for the new review-web relates to our discussion about how we reference the names of private repositories in public repositories. I suggested using nicknames for our modules without mentioning their real names in the public repositories. Although these nicknames are not ideal to include in the codebase, people have started using them anyway.

Meanwhile, you pointed out that the separation between public and private repositories should meet confidentiality criteria if they are properly divided. It turns out that removing the config-related items addresses both your concerns and my intentions. In fact, the removal was made because we switched to using TOML strings for the configuration data format.

Since engineers are already aware that they should use the nicknames and are actively doing so, I hope they will be informed of the new policy as soon as possible, so they will need to make fewer fixes. The key point of the new policy is that there is no longer a need to directly mention the names of private modules in the code itself, and it allows for more flexibility in referring to private modules in non-code text.

**Comment by msk** [comment]
Thanks for the detailed explanation. I’ve already merged the PR, as it addresses both the confidentiality concerns and the need for flexibility in referencing private repositories.

What I wanted to ask is whether the new release is actively blocking any work or progress on the review-web side. If engineers can proceed with version 0.6.0 without issue in the interim, it might be more efficient to hold off on the release until there are additional changes or fixes. However, if this update is crucial for the adoption of the new policies or required for the continued development of review-web, I agree we should prioritize the release. Could you clarify if this is an immediate need for review-web, or can it proceed with releasing a new version based on 0.6.0?

**Comment by sehkone** [comment]
Technically, no work is being blocked because of this. What I wanted was to ensure that this deletion wouldn’t cause any issues throughout our codebase. Once confirmed, I planned to immediately announce the deletion and the new policy on how to refer to our private modules in both code and non-code contexts.

However, since I believe it won’t cause any issues, it’s still fine to inform people that the deletion will take place and have them follow the new policy, which I will announce at the same time. If you prefer this approach, I’ll proceed.

**Comment by msk** [comment]
> However, since I believe it won’t cause any issues, it’s still fine to inform people that the deletion will take place with an upcoming release, and ask them to follow the new policy, which I will announce immediately. If you prefer this approach, I’ll proceed.

Fine with me.

**Comment by msk** [comment]
Resolved in 0.7.0.
